NitroxModel/DataStructures/GameLogic/Entities/Metadata/KeypadMetadata.cs

```diff
@@ -1,30 +1,33 @@
-﻿using System;
+using System;
 using System.Runtime.Serialization;
 using BinaryPack.Attributes;
 
-namespace NitroxModel.DataStructures.GameLogic.Entities.Metadata
+namespace NitroxModel.DataStructures.GameLogic.Entities.Metadata;
+
+[Serializable]
+[DataContract]
+public class KeypadMetadata : EntityMetadata
 {
-    [Serializable]
-    [DataContract]
-    public class KeypadMetadata : EntityMetadata
-    {
-        [DataMember(Order = 1)]
-        public bool Unlocked { get; }
+    [DataMember(Order = 1)]
+    public bool Unlocked { get; }
 
-        [IgnoreConstructor]
-        protected KeypadMetadata()
-        {
-            // Constructor for serialization. Has to be "protected" for json serialization.
-        }
+    [DataMember(Order = 2)]
+    public string PathFromRoot { get; }
```

Comment:
Is this really a metadata ?

Reply:
This metadata is applied to an object which doesn't necessarily hold the KeypadDoorConsole component. Thus we need a way to find the component, which varies from one prefab to another

Reply:
Can we not do this by extracting it from the object it's on rather than using the path?

Reply:
GetComponentInChildren is quite costy and might pick the wrong door is there are two available in the children